### ISSUE TITLE
Quest: Stunning Defeat At The Ring [11300]

### ISSUE THREAD

**Issue by Xiondar** [dalaranwow/dalaran-wow#5847]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: None of the Gladiator mobs are spawning and because of this Oluf will not spawn, making this quest not complete-able.
**Expected behaviour**: There are supposed to be named gladiators that spawn in the Ring of Judgement which is in the Utgarde Catacombs. Once you kill them, the named Mob "Oluf The Violent" should spawn and when you kill him, he should drop the "Ancient Cipher".

**Steps to reproduce the problem**:

1. Travel to Utgarde Catacombs.
2. Go down to the Ring of Judgement.
3. See that mobs are not spawning.

**Include proofs for this behaviour** 

PROOF: https://imgur.com/a/WKOgGcn
WoW Head Quest Page: https://www.wowhead.com/quest=11300/stunning-defeat-at-the-ring

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Xiondar** [comment]
Nevermind, I figured it out. Its still a little buggy. After accepting the quest you have a very short amount of time where the first mob spawns before it despawns. I must have missed it. Ignore and close this please and sorry about that.
